src/app/core/basic-datatypes/number/number-range-filter/number-range-filter.component.scss

```diff
@@ -0,0 +1,12 @@
+div {
+  display: flex;
+}
+input {
+  border: none;
+  background: none;
+  padding: 0;
+  outline: none;
+  font: inherit;
+  text-align: center;
+  color: currentColor;
+}
```

Comment:
are these still needed? It looks like those elements are not in the html template anymore?

Reply:
You oseem to be right. Going to delete empty .scss file then.